requirements.txt

```diff
@@ -24,3 +24,5 @@ typing_extensions
 scikit-learn==1.1.2
 pyarrow
 opencv-python>=4.9
+tensorflow-datasets==4.9.4
+tensorflow==2.15.0
```

Comment:
There are 2 considerations on my end by changing the requirements for the whole project:

1. Use of exact match specifier.
I think this should be discouraged for a better compatibility. In this context my reasonings are:
    - Are these exact version numbers required by google-research's SEAHORSE?
    - Does the specifier indicate that other versions would break?
    - The versions are relatively new release, i.e.: conda-forge version of `tensorflow-datasets` is only [4.8.3](https://anaconda.org/conda-forge/tensorflow-datasets) (as of 27 Feb '24).

2. The necessity of depending on the libraries
I did not give any suggestion on point 1, sorry for that, and it is related to this point.
Some questions I would like to raise:
    - Is it necessary to depend on `tensorflow`?
      IMHO, SEACrowd should not directly depend on any DL Framework.
    - Is there any work-around for not depending on `tensorflow-datasets` to access the dataset?
      i.e.: download the raw dataset and extract it ourselves?
      (Although, I think dependancy on `tensorflow_datasets` is still reasonable)

Reply:
updated accordingly